apps/components-e2e/src/components/chart/selection-area/selection-area.e2e.ts

```diff
@@ -54,7 +54,11 @@ test('should not have an initial selection', async (testController: TestControll
   await testController.expect(await timestampSelection.exists).notOk();
 });
 
-test('should create a range when a selection will be dragged', async (testController: TestController) => {
+// TODO: lukas.holzer investigate why on Browserstack the overlay has outdated information
```

Comment:
Should we reference the issue here?
https://github.com/dynatrace-oss/barista/issues/244